api/MediaRecorderErrorEvent.json

```diff
@@ -0,0 +1,114 @@
+{
+  "api": {
+    "MediaRecorderErrorEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaRecorderErrorEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false,
+            "notes": "Chrome and Opera use a generic event with an <code>error</code> property."
```

Comment:
I propose a version without saying Chrome and Opera (Opera is a bit strange in the context here), but that still allow note collapsing to happen:
"Uses a generic event with an error property instead."